app/Http/Controllers/FaqController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Faq;
+use Illuminate\Http\Request;
+
+class FaqController extends Controller
+{
+    public function index()
```

Comment:
"Missing doc comment" are easy to fix. Just type '/**' + enter in the empty line above the method and your IDE will generate a doc comment block for you